maps/away/away_site/big_derelict/bigderelict.dm

```diff
@@ -2,7 +2,8 @@
 	name = "large derelict"
 	description = "A very large derelict station. According to the starmap, it shouldn't exist."
 	suffixes = list("away_site/big_derelict/bigderelict.dmm")
-	sectors = list(SECTOR_TAU_CETI, SECTOR_ROMANOVICH, SECTOR_CORP_ZONE, SECTOR_VALLEY_HALE, SECTOR_NEW_ANKARA, SECTOR_BADLANDS, SECTOR_AEMAQ, ALL_COALITION_SECTORS)
+	sectors = list(SECTOR_ROMANOVICH, SECTOR_CORP_ZONE, SECTOR_VALLEY_HALE, SECTOR_NEW_ANKARA, SECTOR_BADLANDS, SECTOR_AEMAQ, ALL_COALITION_SECTORS)
+	sectors_blacklist = list(SECTOR_BURZSIA, SECTOR_HANEUNIM)
```

Comment:
Why Burzsia? It's the "ship graveyard and dumping area".

Reply:
this was largely due to the description saying "according to the map, it shouldn't exist". That being said I might change it